### DISCUSSION TITLE
Activamos Discussions

### DISCUSSION THREAD

**Discussion by MacaMorales** [CodeSystem2022/HelloWorld-3erSemestre#21] [General]
Hola chicos vamos activando este espacio, y además vamos discutiendo algunas cositas.

**Comment by Marcos7gomez** [comment]
Buenas, que les parece si vamos creando las semanas que faltan hasta el dia de hoy y que cada uno vaya subiendo sus tareas al ritmo que le sea conveniente para que nos vayamos poniendo al dia. Nos quedan 11 días para ponernos al día. No esperemos hasta último momento

**Reply by MacaMorales** [reply]
Buenas recién veo el mensaje, me parece perfecto.. SI si cada uno va subiendo a medida que pueda. También nos tenemos que poner de acuerdo con la semana que vamos a mostrar en el video, y el proyecto en Python..

**Reply by FranciscoTejada** [reply]
Hola chicos, yo ya estoy preparando la semana 6!

**Reply by Valenttinna** [reply]
Buenas, si me parece perfecto. Acabo de ver que falta una tardea de la Semana 4. Asi la voy cerrando.
